node/cmd/guardiand/node.go

```diff
@@ -666,7 +666,12 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 
 		if err := supervisor.Run(ctx, "ethwatch",
-			ethereum.NewEthWatcher(*ethRPC, ethContract, "eth", common.ReadinessEthSyncing, vaa.ChainIDEthereum, lockC, setC, chainObsvReqC[vaa.ChainIDEthereum], unsafeDevMode, *ethPollIntervalMs).Run); err != nil {
+			ethereum.NewEthWatcher(*ethRPC, ethContract, "eth", common.ReadinessEthSyncing, vaa.ChainIDEthereum, lockC, setC, chainObsvReqC[vaa.ChainIDEthereum], unsafeDevMode, ethPollIntervalMs, false).Run); err != nil {
+			return err
+		}
+
+		if err := supervisor.Run(ctx, "bscwatch",
+			ethereum.NewEthWatcher(*bscRPC, bscContract, "bsc", common.ReadinessBSCSyncing, vaa.ChainIDBSC, lockC, setC, chainObsvReqC[vaa.ChainIDBSC], unsafeDevMode, nil, true).Run); err != nil {
```

Comment:
No, ETH uses an HTTP client to poll blocks, but BSC uses a WebSocket client, I'm not sure why Wormhole uses a different approach for ETH and BSC.

Reply:
Interesting, since in Guardian what we provide for `--ethRPC` in testnet so far is [Websocket](https://github.com/alephium/bridge-infrastructure/blob/wormhole-infra/testnet/docker/guardian-0-vm-docker-compose.yaml#L30) endpoint as well

Reply:
Yeah, I suppose it's because both an HTTP server and a WebSocket server are running on the same URL, so ETH can connect using an HTTP client.